build/ci.sh

```diff
@@ -90,7 +95,7 @@ if [[ $STATIC == true ]]; then
     cd ..
 
     echo "build libxml2"
-    git clone https://gitlab.gnome.org/GNOME/libxml2
+    git clone https://gitlab.gnome.org/GNOME/libxml2 || true
```

Comment:
I'm not sure we want to change this here. We want the build to fail if the git clone fails

Reply:
Makes sense. I added it because the build script was failing for the second and consequent runs on the persistent environment. The git folder remains between runs, so the clone operation fails. And it's probably not the case for the ephemeral CI environments. I could add `rm -rf ./libxml2` instead. WDYT?

Reply:
Maybe just skip checkout if `libxml2` folder exists? I don't like the idea of 'helpfully' removing files